zebra-chain/src/transaction/unmined.rs

```diff
@@ -54,13 +54,13 @@ mod zip317;
 /// > transparent transactions because of their size.
 ///
 /// [ZIP-401]: https://zips.z.cash/zip-0401
-pub const MEMPOOL_TRANSACTION_COST_THRESHOLD: u64 = 4000;
+pub const MEMPOOL_TRANSACTION_COST_THRESHOLD: u64 = 10_000;
```

Comment:
Also update the comment with the new spec and rationale.

Reply:
Thanks, fixed in 0da6b5f0dc